package.json

```diff
@@ -23,6 +23,8 @@
     "lint": "eslint --ext .js,.ts,.tsx,.json .",
     "lint-fast": "eslint --cache --fix --ext .js,.ts,.tsx,.json .",
     "fmt": "prettier --cache --write . && npm run lint -- --fix",
+    "openapi-gen-ts": "openapi-gen-ts",
+    "prettier": "prettier",
```

Comment:
This feels silly but the only other way I know to use the installed versions in the script is to hit `node_modules/.bin/prettier` and `node_modules/.bin/openapi-gen-ts` directly. I guess we could do that?